lighty-applications/lighty-app-modules-config/src/main/java/io/lighty/applications/util/ModulesConfig.java

```diff
@@ -80,9 +80,6 @@ public boolean equals(final Object obj) {
         if (!(obj instanceof ModulesConfig)) {
             return false;
         }
-        if (!super.equals(obj)) {
-            return false;
```

Comment:
Please remove `super` also from hash code method. Rename commit header to "ModulesConfig reference should not be compared" or something similar.

Reply:
super changed to `this`

Reply:
I do not see `this` at all here :D

Reply:
removed